### PR TITLE
Bads per run

### PR DESCRIPTION
**Pull request by SophieHerbst**
allow for definition of bad channels per run
if plot = True, plot single runs

### CODE REVIEW

**Comment by SophieHerbst** [comment]
Two questions: 

1. Should we rename the variable `run`? 
> `for run in config.runs:`

Executing `run` in debug mode breaks the script. 

2. At what stage to concatenate the runs and save one file? In `02-maxfilter` or in `03-extract_events`?

File: README.md [hunk 1]
```diff
@@ -6,42 +6,69 @@ Study analysis template with MNE
...
+
+#### AUTOMATIC REJECTION OF ARTIFACTS
 - `reject` : the default rejection limits to make some epochs as bads. This allows to remove strong transient artifacts. **Note**: these numbers tend to vary between subjects.
+
+#### EPOCHING
 - `tmin`: float that gives the start time before event of an epoch.
 - `tmax` : float that gives the end time after event of an epochs.
 - `baseline` : tuple that specifies how to baseline the epochs. If None, then no baseline applied.
 - `event_id` : python dictionary that maps events (trigger/marker values) to conditions. E.g. `event_id = {'Auditory/Left': 1, 'Auditory/Right': 2}`
 - `runica` : boolean that says if ICA should be used or not.
```
Comment (agramfort) [inline README.md]: Remove as below

File: config.py [hunk 2]
```diff
@@ -30,43 +30,52 @@
...
-# naming should be consistant over subjects
-# put the number of runs you ideally expect to have per subject
-# the scripts will issue a warning if there are less
-# leave empty if there is just one file
-runs =  [''] # ['run01', 'run02']
+# The MEG-data need to be stored in a folder
+# named my_study_path/MEG/my_subject/
 
+# This is the name of your experimnet
+study_name = 'audvis'
```
Comment (agramfort) [inline config.py]: Where do you use it?

**Comment by agramfort** [comment]
>
> Two questions:
>
>    1. Should we rename the variable run?
>
> for run in config.runs:
>
>
run is the naming used by BIDS-MEG. See:

https://hal.archives-ouvertes.fr/hal-01591080/document

and look for what we mean by session and run.

> Executing run in debug mode breaks the script.
>
how does it break it?

>
>    1. At what stage to concatenate the runs and save one file? In
>    02-maxfilter or in 03-extract_events?
>
> I would not concatenate the raws but just at the epochs level. otherwise
it's going to require too much memory I fear.

File: config.py (hunk 2 above)
Comment (SophieHerbst) [reply config.py]: `base_raw_fname = '{subject}_' + study_name + '{run}_raw.fif'`

**Comment by SophieHerbst** [comment]
@agramfort, when I put a breakpoint in the function and then execute ` run`  (to check which run I am in) from the command line, it says: 

```

  File "/Users/sophie/anaconda/envs/mne/lib/python3.6/pdb.py", line 1028, in do_run
    raise Restart

Restart
```

**Comment by agramfort** [comment]
from ipython the magic command should be

%run

not just

run

>

**Comment by SophieHerbst** [comment]
Hm. Ok. Not sure how to fix this. 
But if it is not a general problem, let's keep run.

**Pull request merged**